### ISSUE TITLE
BackendFlagsRendererHasVtxOffset flag is not working

### ISSUE THREAD

**Issue by Cardroid** [AllenDang/cimgui-go#236]
If the size of the data is larger than a certain size, there is a problem that the ImPlot does not draw.

https://github.com/AllenDang/cimgui-go/assets/25889754/db066ac4-6934-4d5c-b84d-0ec65f928893

It's part of my initialization code.
(I referred to https://github.com/ocornut/imgui/issues/2591)

```go
func InitImgui(title string, width, height int) {
	Context = &ImguiWContext{}
	Context.context = imgui.CreateContext()

	io := imgui.CurrentIO()

	// io.SetConfigViewportsNoAutoMerge(true)
	io.SetConfigFlags(imgui.ConfigFlagsDpiEnableScaleViewports)
	io.SetBackendFlags(imgui.BackendFlagsRendererHasVtxOffset)

	io.SetIniFilename("")

	imgui.PlotCreateContext()
	imgui.ImNodesCreateContext()

	Context.imBackend = imgui.CreateBackend(imgui.NewGLFWBackend())

	// logger.Tracef("imgui.CurrentIO().IniFilename()=%v", imgui.CurrentIO().IniFilename())
	// logger.Tracef("io.IniFilename()=%v", io.IniFilename())

	Context.imBackend.SetTargetFPS(60)
	Context.imBackend.SetBeforeRenderHook(beforeRender)
	Context.imBackend.SetAfterCreateContextHook(afterCreateContext)
	Context.imBackend.SetBeforeDestroyContextHook(beforeDestroyContext)
...
```

The flag does not appear to be working properly.
I'd appreciate your help :)

**Comment by AllenDang** [comment]
What's your giu version?

**Comment by Cardroid** [comment]
I'm sorry that I didn't put any important information on it.

I'm only using `cimgui-go` because of the bug in `giu`
And I'm currently using the code on [ptxmac/cimgui-go/tree/ptx_fixes](https://github.com/ptxmac/cimgui-go/tree/ptx_fixes) due to a problem that cimgui-go is not building on Windows (#206)

**Comment by Cardroid** [comment]
The reason was that the opengl version was 3.0.
It works well when it is fixed at 3.2.
